### PR TITLE
Epic 2/story set buildings 5

### PR DESCRIPTION
**Pull request by Weigiu**
select building as Start or Destination

### CODE REVIEW

**Comment by samiha-rahman** [comment]
continuing work on #112

**Comment by Weigiu** [comment]
commit e25d5c2 linked to #126 **

File: SD-K9/src/app/components/building-info/building-info.component.html [hunk 1]
```diff
@@ -5,6 +5,14 @@
     <img tappable src="../../../assets/images/close-icon.png" class="close-card"
       (click)="hideBuildingInfoCard($event)" />
   </ion-card-header>
+  <ion-row>
+     <ion-col>
+      <ion-button  color="primary" class="location-btn" (click)="setStartLocation()" style="width:100%">Set Start</ion-button>
+    </ion-col>
+    <ion-col>
+      <ion-button color="primary" class="location-btn" (click)="setDestionationLocation()" style="width:100%">Set Destination</ion-button>
```
Comment (samiha-rahman) [inline SD-K9/src/app/components/building-info/building-info.component.html:13]: setDestinationLocation() spelled wrong

File: SD-K9/src/app/components/building-info/building-info.component.ts [hunk 2]
```diff
@@ -76,6 +78,15 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
       this.buildingInfoCardIsShown = false;
     }
   }
+  setStartLocation(){
+    let fullname=this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['fullName'];
```
Comment (samiha-rahman) [inline SD-K9/src/app/components/building-info/building-info.component.ts:82]: use camel case!

File: SD-K9/src/app/components/building-info/building-info.component.ts [hunk 3]
```diff
@@ -79,13 +79,13 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
     }
   }
   setStartLocation(){
-    let fullname=this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['fullName'];
-    this._directionFormStore.setSource(fullname);
+    let fullName=this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['fullName'];
+    this._directionFormStore.setSource(fullName);
   }
 
   setDestionationLocation(){
```
Comment (samiha-rahman) [inline SD-K9/src/app/components/building-info/building-info.component.ts:86]: You forgot to fix the method name here

File: SD-K9/src/app/components/building-info/building-info.component.ts [hunk 4]
```diff
@@ -102,7 +113,9 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
 
   hideBuildingInfoCard(event) {
     this.buildingInfoCardIsShown = false;
-    this._buildingInfoStore.clearBuildingInfo();
+
+    //Makes clickable buildings more reliable
+    //this._buildingInfoStore.clearBuildingInfo();
```
Comment (ThierryRomain) [inline SD-K9/src/app/components/building-info/building-info.component.ts:118]: Should delete if not useful

Comment (Weigiu) [reply SD-K9/src/app/components/building-info/building-info.component.ts:118]: I'll uncomment it since it's to makde hideBuildingInfoCard for reliable

**Pull request merged**